internal/pwschemes/argon2i/argon2i.go

```diff
@@ -0,0 +1,32 @@
+package argon2i
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+
+	"golang.org/x/crypto/argon2"
+)
+
+const (
+	time    uint32 = 4
+	memory  uint32 = 32 * 1024
```

Comment:
As per section 8 and 9 of the [Argon2 paper](https://github.com/P-H-C/phc-winner-argon2/blob/master/argon2-specs.pdf), I think we might want to increase this, unless it's the settings used by Dovecot?

Reply:
No, these values are encoded into the hash and Dovecot will re-use whatever we use here. I haven't read the whole paper so I'm not sure if I understand those correctly, yet. But I'll start with a little more and we can change it or make it configurable later.